cpp/velox/shuffle/VeloxSplitter.h

```diff
@@ -105,47 +146,49 @@ class VeloxSplitter {
 
   // for debugging
   void PrintColumnsInfo() const {
-    PRINT_FUNCTION_SPLIT_LINE();
-    PRINTLF(fixed_width_column_count_);
+    VS_PRINT_FUNCTION_SPLIT_LINE();
```

Comment:
When will we delete the debug functions? Or keep it for further use?

Reply:
I want to keep these functions for further debug

Reply:
Would you agree? @FelixYBW And do we need to keep file  Print.h?

Reply:
it's convenient for our debugging and I use a macro to open or close it.
I think we need some helper functions to make Print more easy.

Reply:
otherwise, you have to use std::cout to output some information manual, writing the same code repeatly